client/src/features/ProjectPageV2/ProjectPageContent/DataSources/DataSourceDisplay.tsx

```diff
@@ -214,79 +183,42 @@ export function DataSourceDisplay({
     setToggleView((open: boolean) => !open);
   }, []);
 
-  const nameDisplay = (
-    <div
-      className={cx(
-        "d-flex",
-        "align-items-center",
-        "gap-1",
-        "cursor-pointer",
-        "ps-2",
-        sessionItemStyles.ItemDisplaySessionName
-      )}
-      onClick={toggleDetails}
-    >
-      <LayoutSidebarInsetReverse
-        className={cx("flex-shrink-0", "me-0", "me-sm-2")}
-        size="20"
-      />
-      <span className={cx("text-truncate")} data-cy="data-storage-name">
-        {storageSensitive.name}
-      </span>
-    </div>
-  );
+  const storageType = storageSensitive?.storage_type ? (
+    <span className="fst-italic" data-cy="data-storage-type">
+      (type: {storageSensitive?.storage_type})
+    </span>
+  ) : null;
```

Comment:
Avoid rendering the space `" "` when this is `null`:

```suggestion
  const storageType = storageSensitive?.storage_type ? (
    <>
      {" "}
      <span className="fst-italic" data-cy="data-storage-type">
        (type: {storageSensitive.storage_type})
      </span>
    </>
  ) : null;
```